types/canonical_json.go

```diff
@@ -1,11 +1,16 @@
 package types
 
 import (
+	"time"
+
 	"github.com/tendermint/go-wire/data"
 )
 
 // canonical json is go-wire's json for structs with fields in alphabetical order
 
+// timeFormat is RFC3339Millis, used for generating the sigs
+const timeFormat = "2006-01-02T15:04:05.999Z07:00"
```

Comment:
can we take this from go-wire instead? ie.  wire.RFC3339Millis

Reply:
okay